apache/incubator-zeppelin
# ZEPPELIN-815 don't create a sub shell for the runner

### What is this PR for?

This pull request is supposed to fix ZEPPELIN-815. The issue is that one cannot stop Zeppelin by sending a signal, because it starts the runner in a sub shell. The pull request starts the runner in the same process as the `zeppelin.sh`, making it react to signals.

### What type of PR is it?

Improvement

### Todos

None

### What is the Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-815

### How should this be tested?
- Start `zeppelin.sh`
- Get PID of the `zeppelin.sh` process
- Send SIGINT (`kill -SIGINT <pid>`) to the `zeppelin.sh` process
- Observe that Zeppelin is stopped

### Screenshots

none

### Questions:
* What is the reason to put an exec into a sub shell in the first place?
